### ISSUE TITLE
Misleading TODO statement in notebook 9.2

### ISSUE THREAD

**Issue by girdeux31** [udlbook/udlbook#140]
I think the second TODO in the last cell is misleading:

```
# TODO:  Run the gradient descent on the modified loss
# function with 100 steps and a very small learning rate alpha of 0.05
# Replace this line:
grad_path_typical_lr = None ;
```

It refers to the **modified** loss function. But, according to the last sentence, it should be the GD on the **original** loss. So, something like this would be more appropriate:

`# TODO:  Run the gradient descent on the original loss function`

**Comment by udlbook** [comment]
Thanks so much Carles.  Fixed!  Sorry that this took me so long.  I'm quite behind on fixing things!
